contracts/ETO/ETOCommitment.sol

```diff
@@ -916,58 +927,57 @@ contract ETOCommitment is
         // write in single pack, hopefully storage will be optimized...
         ticket.equivEurUlps = uint96(newEquivEurUlps);
         ticket.rewardNmkUlps = newRewardNmkUlps;
-        ticket.equityTokenInt = newEquityTokenInt;
+        ticket.equityTokenAmount = newEquityTokenAmount;
         ticket.amountEth = newAmountEth;
         ticket.amountEurUlps = newAmountEurUlps;
         ticket.usedLockedAccount = usedLockedAccount;
     }
 
     function updateTotalInvestment(
-        uint256 equityTokenInt,
-        uint256 fixedSlotEquityTokenInt,
+        uint256 equityTokenAmount,
+        uint256 fixedSlotEquityTokenAmount,
         uint256 equivEurUlps,
         uint256 amount,
         bool firstTimeInvestment
     )
         private
     {
-        // total number of tokens must fit into uint56, uints are coerced to biggest size
-        uint256 newTotalTokensInt = _totalTokensInt + equityTokenInt;
-        assert(newTotalTokensInt < 2**56);
-        // as fixedSlotEquityTokenInt256 always lte equityTokenInt no chances to overflow
```

Comment:
I dont get this comment, nor the new one.
new code is: 
uint 128 = uint128  + uint128

why can this not overflow?